src/test/java/com/marklogic/appdeployer/command/restapis/DeleteRestApiTest.java

```diff
@@ -50,6 +54,28 @@ public void createAndDelete() {
         assertFalse(serverMgr.exists(SAMPLE_APP_NAME), "The REST API app server have been deleted");
     }
 
+	@Test
```

Comment:
This test can't succeed until the bug is fixed. And if it's run, it leaves behind a REST API app server that cannot be deleted via "/v1/rest-apis" and has to be deleted via the Admin UI.